common/src/main/java/com/microsoft/identity/common/internal/fido/WebAuthnJsonUtil.kt

```diff
@@ -87,8 +93,16 @@ class WebAuthnJsonUtil {
                 WEBAUTHN_RESPONSE_CLIENT_DATA_JSON_KEY))
             assertionResult.put(WEBAUTHN_RESPONSE_SIGNATURE_JSON_KEY, authResponseJsonObject.get(
                 WEBAUTHN_RESPONSE_SIGNATURE_JSON_KEY))
-            assertionResult.put(WEBAUTHN_RESPONSE_USER_HANDLE_JSON_KEY, authResponseJsonObject.get(
-                WEBAUTHN_RESPONSE_USER_HANDLE_JSON_KEY))
+            try {
+                assertionResult.put(
+                    WEBAUTHN_RESPONSE_USER_HANDLE_JSON_KEY, authResponseJsonObject.get(
+                        WEBAUTHN_RESPONSE_USER_HANDLE_JSON_KEY
+                    )
+                )
+            } catch (e: JSONException) {
```

Comment:
>catch (e: JSONException) { [](http://example.com/codeflow?start=14&length=26)

Is there a reason why you're not checking user handle's presence and going through exception catching?

consider adding exception message or isNull(authResponseJsonObject.get(WEBAUTHN_RESPONSE_USER_HANDLE_JSON_KEY))

Reply:
Updated: aa0d05a3019d16a222957d9f45a0105002745b01